packages/combo-box/test/internal-filtering.test.js

```diff
@@ -238,16 +251,28 @@ describe('internal filtering', () => {
     });
   });
 
-  describe('setting items when opened', () => {
+  describe('setting filtered items when opened', () => {
     beforeEach(() => {
+      comboBox = fixtureSync('<vaadin-combo-box></vaadin-combo-box>');
       comboBox.items = [];
-    });
-
-    it('should properly display all items in the selector', () => {
       comboBox.open();
       comboBox.filteredItems = makeItems(10);
       flush();
+    });
+
+    it('should properly display all items in the selector', () => {
       expect(getAllItems(comboBox).length).to.equal(10);
     });
   });
+
+  describe('setting placeholder items', () => {
```

Comment:
**note:** This is a new test that is intended to cover the regression discovered by the Flow ITs.